docs/cookbook/payments/stripe.rst

```diff
@@ -1,6 +1,21 @@
+.. warning::
+
+    On September 14, 2019 the `Strong Customer Authentication (SCA)
+    <https://eur-lex.europa.eu/legal-content/EN/TXT/uri=uriserv:OJ.L_.2018.069.01.0023.01.ENG&toc=OJ:L:2018:069:TOC>`_
+    requirement has been introduced.
+    The implementation provided by Sylius core was not *SCA Ready* and has been deprecated.
+    Please have a look at the [official documentation of Stripe regarding this topic](https://stripe.com/guides/strong-customer-authentication).
```

Comment:
```suggestion
    Please have a look at the (official documentation of Stripe regarding this topic)<https://stripe.com/guides/strong-customer-authentication>.
```

Didn't the formattion look suspicious to you? 🎉

Reply:
Sorry my mistake 🎃